WordPress/Classes/ViewRelated/Reader/ReaderCommentsViewController.m

```diff
@@ -184,6 +185,27 @@ - (void)willRotateToInterfaceOrientation:(UIInterfaceOrientation)toInterfaceOrie
     [self.tableViewHandler refreshCachedRowHeightsForWidth:width];
 }
 
+- (void)viewWillTransitionToSize:(CGSize)size withTransitionCoordinator:(id<UIViewControllerTransitionCoordinator>)coordinator
+{
```

Comment:
Considering all the internal view hierarchy changes between 7 and 8, I'm not surprised at all. :p